eos-add-flatpak-gnome-repos.service

```diff
@@ -0,0 +1,18 @@
+# Add the stable GNOME flatpak repositories after an OS upgrade
+
+[Unit]
+Description=Configure the stable GNOME flatpak repositories
+DefaultDependencies=no
+Conflicts=shutdown.target
```

Comment:
I think you want to add `After=local-fs.target` and `Wants=local-fs.target`. This would fail if systemd decided to run it before all the filesystems were mounted.